test/integration/auth/node_test.go

```diff
@@ -254,7 +254,7 @@ func TestNodeAuthorizer(t *testing.T) {
 			startTime := metav1.NewTime(time.Now())
 			_, err := client.CoreV1().Pods("ns").UpdateStatus(context.TODO(), &corev1.Pod{
 				ObjectMeta: metav1.ObjectMeta{Name: "node2normalpod"},
-				Status:     corev1.PodStatus{StartTime: &startTime},
+				Status:     corev1.PodStatus{StartTime: &startTime, QOSClass: corev1.PodQOSBestEffort},
```

Comment:
let's not mix in all other changes into this PR. If we want to improve all these tests, we can do it in separate PR. Let's concentrate this PR to an API validation change